src/pages/GetHelp.vue

```diff
@@ -294,6 +354,11 @@ export default {
         this.loading = false
         this.error = e
       }
+    },
+
+    fromToday(date) {
+      const today = new Date().toLocaleDateString().split('/')
+      return date >= today[2] + '/' + today[1] + '/' + today[0]
```

Comment:
this also looks silly, if the quasar date api doesn't have nice date manipulation functions, then maybe look into https://date-fns.org/